src/api/docgen/view/attribute.ts

```diff
@@ -0,0 +1,14 @@
+import * as reflect from "jsii-reflect";
+import { Markdown } from "../render/markdown";
+import { Transpile } from "../transpile/transpile";
+import { Property } from "./property";
+
+export class Attribute {
```

Comment:
Add some docstrings

P.S. what is an attribute? I am not familiar with that term in the JSII world

Reply:
The current python docs use `Attributes` for instance properties (i.e `instance.some_property`).

In practice there is no different between this and a `reflect.Property` - which is why it simply delegates to the `Property` class. I'm quite font of this distinction because we normally use `Properties` to mean the input properties of a construct, not the output ones.

Reply:
But isn't that a Python-specific terminology.

(we use "props" and "properties" to distinguish between initialization props and class properties -- not that I like it but that's the semantic we used so far).

Reply:
Also, the term "attribute" is used in the AWS CDK to denote resource attributes (such as ARN), so that might also be conflicting..

Reply:
I'd go with `Property` here and if you want a term for the initialization props use `prop`

Reply:
Ok, i'll just reuse the `Property` class for now and we'll see where that gets us. 

This also brings up the topic of having language specific header titles (i.e `Attributes` in python and `Properties` in TS) which my current design doesn't address. I'll use `Properties` for now since I agree it is probably more generic.